elements/rh-card/demo/demo.css

```diff
@@ -50,11 +50,11 @@ rh-card.full img {
 
 rh-card.alt,
 rh-context-provider[color-palette^="light"] rh-card.alt {
-  --rh-card-background-color: #f0f0f0;
+  --rh-background-color: #f0f0f0;
```

Comment:
we don't have an existing token `--rh-background-color`. perhaps you meant `--rh-card-background-color`?

does this need to be a custom property? could we just use built-in property `background-color`?